app/src/main/res/layout/add_meeting_form.xml

```diff
@@ -88,7 +88,7 @@
         android:layout_height="65dp"
         android:layout_marginTop="272dp"
         android:ems="10"
-        android:hint="Location/Link"
+        android:hint="@string/locationLink"
```

Comment:
Utilized string.xml instead of hardcoded strings. This is best practice for localization and organization.

Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://developer.android.com/guide/topics/resources/string-resource